src/index.spec.ts

```diff
@@ -439,11 +448,12 @@ describe('Try batch operations', () => {
 
     const w1 = tree.get(Buffer.from('a'));
     const w2 = tree.get(Buffer.from('b'));
-    const w3 = tree.get(Buffer.from('c'));
+    const w3 = tree.rlpSerializeWitness(tree.get(Buffer.from('c')));
```

Comment:
Ok, this is why this is confusing. Why are you putting rlpSerializeWitness here? Just do it below so that the style is consistent.